groups/restrictions.yaml

```diff
@@ -145,6 +145,7 @@ restrictions:
       - "^k8s-infra-staging-nfd@kubernetes.io$"
       - "^k8s-infra-staging-npd@kubernetes.io$"
       - "^k8s-infra-staging-sp-operator@kubernetes.io$"
+      - "^k8s-infra-staging-node-ipam-controller@kubernetes.io$"
```

Comment:
it has to go in  sig-network/groups, the previous block

Reply:
Thanks, I put the wrong side. Updated.